Device Triggers UI visitbility note added (#20816)
home-assistant/home-assistant.github.io: source/_integrations/homekit.markdown

# Device Triggers

> Devices that support triggers can be added to the bridge by accessing options for the bridge in **{% my integrations title="Configuration >> Integrations" %}**. Note that the Device Triggers configuration is only visible if Advanced Mode is enabled in your profile. 

Reviewer: ```suggestion
Devices that support triggers can be added to the bridge by accessing options for the bridge in **{% my integrations title="Configuration >> Integrations" %}**. To use this feature, Advanced Mode must be enabled in your user profile.
```
